shell/shared/demo.js

```diff
@@ -71,15 +79,20 @@ if (Meteor.isServer) {
 
   if (allowDemo) {
     Meteor.methods({
-      createDemoUser: function (displayName) {
-        // This is a login method that creates a new temporary user every time it is used.
-
+      createDemoUser: function (displayName, isAppDemoUser) {
+        // This is a login method that creates a new temporary user
+        // every time it is used.
+        //
+        // isAppDemoUser is important for stats; see
+        // cleanupExpiredUsers().
         check(displayName, String);
```

Comment:
Please add a check() for `isAppDemoUser`'s type. Granted, your code will deal reasonably with incorrect types due to your `!!`, but as a rule we should always add check()s because it's easy to have Mongo injection bugs otherwise.